File: cpp/bc1181.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
	int li;
	float res = 0.0, ma[12][12];
	char op;
	
	cin >> li >> op;
	
	for(int i = 0; i < 12; i++)
	{
		for(int j = 0; j < 12; j++)
			cin >> ma[i][j];
	}
	
	for(int k = 0; k < 12; k++)
		res += ma[li][k];
	if(op == 'm' || op == 'M') res = res / 12.0;
	
	cout << fixed << setprecision(1) << res << endl;
}

File: python/bc1181.py
ma = [[None for j in range(12)] for i in range(12)]
res = 0.0

li = int(input())
op = str(input())

for i in range(0, 12):
    for j in range(0, 12):
        val = float(input())
        ma[i][j] = val

for k in range(0, 12):
    res += ma[li][k]

if op == "S":
    print(f"{res:.1f}")
else:
    res = res / 12
    print(f"{res:.1f}")
